litestar/middleware/session/client_side.py

```diff
@@ -156,38 +173,23 @@ async def store_in_message(self, scope_session: ScopeSession, message: Message,
 
         scope = connection.scope
         headers = MutableScopeHeaders.from_message(message)
-        cookie_keys = self.get_cookie_keys(connection)
+        connection_cookies = self.get_cookie_key_set(connection)
+        response_cookies: set[str] = set()
 
         if scope_session and scope_session is not Empty:
             data = self.dump_data(scope_session, scope=scope)
-            cookie_params = dict(
-                extract_dataclass_items(
-                    self.config,
-                    exclude_none=True,
-                    include={f for f in Cookie.__dict__ if f not in ("key", "secret")},
-                )
-            )
```

Comment:
Every time we set or clear a cookie, we were iterating over the `Cookie` type, and iterating over the fields of `self.config`. AFAICT, these are static things that shouldn't be changing at all once the app is started up, so continually re-evaluating these seems inefficient.